blocks/library/cover-image/editor.scss

```diff
@@ -22,4 +22,16 @@
 	&.components-placeholder h2 {
 		color: inherit;
 	}
+
+	&.has-left-content {
+		.block-editable__inline-toolbar{
+			justify-content: flex-start;
+		}
+	}
+
+	&.has-right-content {
+		.block-editable__inline-toolbar{
+			justify-content: flex-end;
+		}
+	}
```

Comment:
Minor, but we don't really need the nesting here `(&.has-left-content .block-editable__… {`).